### PR TITLE
(Deposit/Withdraw) Support connection error transfer status in transfer UI

### PR DESCRIPTION
**Pull request by jonator**
### Linear Task

[Linear Task URL](https://linear.app/osmosis/issue/FE-622/support-connection-error-transfer-status-in-transfer-ui)

## Brief Changelog

* Migrate to transfer status type from bridge package in UI, adapt to legacy IBC transfer status history store
* Minor cleanup of interchain balances tRPC query

## Testing and Verifying

Transfer statuses are visible in table, including connection error types

### CODE REVIEW

**Review by JoseRFelix — CHANGES_REQUESTED** [review]

File: packages/web/components/swap-tool/__tests__/swap-tool.spec.tsx [hunk 1]
```diff
@@ -1,241 +0,0 @@
-/* eslint-disable import/no-extraneous-dependencies */
```
[the remark this answers is not in the document]
Comment (jonator) [reply packages/web/components/swap-tool/__tests__/swap-tool.spec.tsx:1]: It started flaking

[the remark this answers is not in the document]
Comment (JoseRFelix) [reply packages/web/components/swap-tool/__tests__/swap-tool.spec.tsx:1]: This test is quite important as it's a smoke test for our swap tool and has helped me identify errors in the past. Do you know which assertion is failing so I can take a look?

[the remark this answers is not in the document]
Comment (jonator) [reply packages/web/components/swap-tool/__tests__/swap-tool.spec.tsx:1]: It was failing on assertion at line 195. What's strange, is sometimes the GitHub action tests succeeds and it fails locally on the same branch. Also, why are pools being queried? It should be using SQS now for quotes.

**Comment by jonator** [comment]
@JoseRFelix what's the plan with fixing the swap tool test? It's worth noting that this is getting replaced by new trade flow very soon as well.

**Comment by JoseRFelix** [comment]
@jonator I approved the PR. By the way, the build is failing because the chainlist added an unexpected property. @JeremyParish69 is working on it

**Pull request merged**